README.md

```diff
@@ -36,339 +34,172 @@
   <a href="https://github.com/returntocorp/semgrep/issues/new/choose">
     <img src="https://img.shields.io/badge/issues-welcome-green?style=flat-square" alt="Issues welcome!" />
   </a>
-      <a href="https://github.com/returntocorp/semgrep/actions?query=workflow%3ATests+branch%3Adevelop">
-      <img src="https://github.com/returntocorp/semgrep/workflows/Tests/badge.svg?branch=develop" alt="Test Status"/>
-  </a>
-  <a href="https://twitter.com/intent/follow?screen_name=r2cdev">
-    <img src="https://img.shields.io/twitter/follow/r2cdev?label=Follow%20r2cdev&style=social&color=blue" alt="Follow @r2cdev" />
+  <a href="https://github.com/returntocorp/semgrep#readme">
+    <img src="https://img.shields.io/github/stars/returntocorp/semgrep?label=GitHub%20Stars&style=flat-square" alt="1000+ GitHub stars" />
   </a>
 </p>
 
-Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.dev/editor).
+Semgrep tl;dr:
 
-Semgrep combines the convenient and iterative style of `grep` with the powerful features of an Abstract Syntax Tree (AST) matcher and limited dataflow. Easily find function calls, class or method definitions, and more without having to understand ASTs or wrestle with regexes.
+- A simple, customizable, and fast static analysis tool for finding bugs
+- Combines the speed and customization of `grep` with the precision of traditional static analysis tools
+- No painful DSL; patterns look like the source code you’re targeting
+- Batteries included with hundreds of existing community rules for OWASP 10 issues and common mistakes
+- Run it in CI, at pre-commit, or in the editor
+- Runs offline on uncompiled code
 
-Visit [Installation](#installation) and [Usage](#usage) to get started.
+Semgrep supports:
 
-## Installation
```

Comment:
Missing JSON and OCaml

Reply:
The construction icon is unclear IMO. Would prefer we say "beta" explicitly.

Reply:
Any reason not to add those two @dlukeomalley ?

Reply:
Nope, hence the note. Just a copy paste bug for me to fix